src/estimation/include/iDynTree/Estimation/BerdyHelper.h

```diff
@@ -128,7 +128,10 @@ enum BerdySensorTypes
     /**
      * Non-physical sensor that measures the wrench trasmitted by a joint.
      */
-    JOINT_WRENCH_SENSOR     = 1003
+    JOINT_WRENCH_SENSOR     = 1003,
+
+    // CoM Accelerometer sensor
+    COM_ACCELEROMETER_SENSOR = 1004
```

Comment:
It'd be great if you could document this sensor with the appropriate measurement model and how it affects the current measurement serialization?

Reply:
@prashanthr05 I updated the main issue with the details related to the measurements handling https://github.com/robotology/idyntree/issues/570#issuecomment-530347743
Please check it and see if any other additional information is needed